docs/fdc3-compliance.md

```diff
@@ -0,0 +1,51 @@
+---
+id: fdc3-compliance
+title: FDC3 Specification Compliance 
+sidebar_label: FDC3 Compliance
+---
+
+
+
+FDC3 standards follow the IETF best practices for Key words to Indicate Requirement levels: [RFC 2119](https://tools.ietf.org/id/draft-faltstrom-uri-11.html#RFC2119).  Documentation should be updated as needed to reflect this.
+
+In general, the ratified FDC3 specs represent a lowest common denominator interface for interoperability.  So, unless a particular item in a spec is marked with keywords such as  OPTIONAL,  MAY, SHOULD, or SHOULD NOT, it should be treated as REQUIRED.  SinceFDC3 itself is primarily concerned with establishing the baseline requirements for interoperation, this is consistent with the IETF Guidance:
+
+>6. **Guidance in the use of these Imperatives**
+>
+>   Imperatives of the type defined in this memo must be used with care
+>   and sparingly.  In particular, they MUST only be used where it is
+>   actually required for interoperation or to limit behavior which has
+>   potential for causing harm (e.g., limiting retransmisssions)  For
+>   example, they must not be used to try to impose a particular method
+>  on implementors where the method is not required for
+>   interoperability.
+
+These rules would apply only to standards work within FDC3.  Today, this covers API, App Directory, Context Data, and Intents working groups, it does not apply to the Use Cases working group. 
+
+### Personas:
+FDC3 implementors generally fall into 2 categories: platform providers, and application providers.  
+A platform provider provides an implementation of the FDC3 API for applications to use. Implicitly, it connects to one or more App Directories.
+
+An application provider is largely a downstream consumer of FDC3 standards.  It MAY use the API, it MAY use Context Data, it MAY use Intents.  Application providers are only required to comply with the standards they make use of.  
+
+Depending on persona, implementation compliance with FDC3 will mean different things.  
+
+#### Platform Provider
```

Comment:
Make this a level 3 heading which will then show as a sub-heading in the navigation guide.